Sources/SwiftDocC/Semantics/Redirect.swift

```diff
@@ -12,16 +12,19 @@ import Foundation
 import Markdown
 
 
-/// A directive that specifies an additional URL for the page where the directive appears.
+/// A directive that specifies a previous URL for the page where the directive appears.
 ///
-/// Use this directive to declare a URL where a piece of content was previously located.
-/// For example, if you host the compiled documentation on a web server,
-/// that server can read this data and set an HTTP "301 Moved Permanently" redirect from
-/// the declared URL to the page's current URL and avoid breaking any existing links to the content.
+/// If the page has moved more than once you can add multiple  `Redirected` directives, each specifying one previous URL.
```

Comment:
```suggestion
/// If the page has moved more than once you can add multiple  `Redirected` directives, each specifying one previous URL. For example:
///
/// ```
/// @Redirected(from: "old/path/to/this/page")
/// @Redirected(from: "another/old/path/to/this/page")
/// ```
```

Reply:
Thanks. I like the examples but I wanted the code block to use syntax highlighting so I applied the changes manually in https://github.com/apple/swift-docc/pull/811/commits/a87ad1dd3379179c437b7be1a048f39c5259e6ef